justfile

```
@@ -641,9 +646,69 @@ archive variant=default_variant kind="repo":
     buildah push "${commit}" "docker://${image}:${version}.${buildid}.${git_commit}.${kind}"
     just fix-ownership
 
+create-folders:
+    #!/bin/bash
+    set -euxo pipefail
+
+    mkdir -p iso
+    mkdir -p repo
+    mkdir -p iso
+
 fix-ownership:
     #!/bin/bash
     set -euxo pipefail
 
-    if [ $SUDO_USER ]; then user="$SUDO_USER"; else user="$(whoami)"; fi
+    just create-folders
+
+    if [ "$EUID" -eq 0 ]; then
+        user="root"
+    else
+        if [ $SUDO_USER ]; then user="$SUDO_USER"; else user="$(whoami)"; fi
+    fi
     chown -R ${user}:${user} iso release repo
+
+gen-secureboot-ephemeral-key:
+    #!/bin/bash
+    set -uxo pipefail
+
+    mkdir -p secureBoot/cfg
+    touch secureBoot/cfg/PK.cfg
+    
+    echo "[ req ]
+    default_bits         = 4096
+    encrypt_key          = no
+    string_mask          = utf8only
+    utf8                 = yes
+    prompt               = no
+    distinguished_name   = my_dist_name
+    x509_extensions      = my_x509_exts
+ 
+    [ my_dist_name ]
+    commonName           = APSensing Platform Key
+    emailAddress         = info@apsensing.com
+
+    [ my_x509_exts ]
+    keyUsage             = digitalSignature
+    extendedKeyUsage     = codeSigning
+    basicConstraints     = critical,CA:FALSE
+    subjectKeyIdentifier = hash" > secureBoot/cfg/PK.cfg
+
+    cp PK.cfg secureBoot/cfg/PK.cfg
+
+    openssl req -x509 -sha256 -days 5490 -outform PEM \
+        -config secureBoot/cfg/PK.cfg \
+        -keyout secureBoot/PK.key -out secureBoot/PK.pem
```

```suggestion
    openssl req -x509 -sha256 -days 5490 -outform PEM -config secureBoot/cfg/PK.cfg -keyout secureBoot/PK.key -out secureBoot/PK.pem
```

I prefer single line statements since we have the hspace for it. Please fix it for all others as well.

---

Done